dev.yml

```diff
@@ -5,7 +5,7 @@ type: nodejs
 up:
   - node:
       yarn: true
-      version: 18.15.0
+      version: 20.9.0
```

Comment:
Bumped while troubleshooting. Might not have been necessary, but it now matches Ruby LSP.